cfspopcon/formulas/scrape_off_layer/two_point_model/two_point_model_algorithms.py

```diff
@@ -45,17 +52,24 @@ def two_point_model_fixed_fpow(
         average_ion_mass: :term:`glossary link<average_ion_mass>`
         kappa_e0: :term:`glossary link<kappa_e0>`
         SOL_momentum_loss_function: :term:`glossary link<SOL_momentum_loss_function>`
+        parallel_conduction_model :term:`glossary link<parallel_conduction_model>`
+        flux_limit_factor_alpha: term:`glossary link<flux_limit_factor_alpha>`
         sheath_heat_transmission_factor: :term:`glossary link<sheath_heat_transmission_factor>`
         two_point_model_error_nonconverged_error: Raise an error if solve does not converge
 
     Returns:
-        :term:`separatrix_electron_temp`, :term:`target_electron_density`, :term:`target_electron_temp`, :term:`target_electron_flux`, :term:`target_q_parallel`,
+        :term:`separatrix_electron_temp`, :term:`target_electron_density`, :term:`target_electron_temp`, :term:`target_electron_flux`, :term:`target_q_parallel`, :term:`kappa_e0', :term:'sheath_heat_transmission_factor', :term:`Spitzer_conduction_reduction_factor', :term:`upstream_SOL_collisionality', :term:`delta_electron_sheath_factor',
```

Comment:
@hassec can you think of a neat way to structure this algorithm so we don't return meaningless values when running without kinetic corrections?